package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pyviz/jupyterlab_pyviz",
-  "version": "0.7.1",
+  "version": "0.7.2",
```

Comment:
Was this previously out of date?  It seems to match the current released version now, not the upcoming release?

Reply:
pyviz_comms and the JLab extension are different packages with different versions.

Reply:
Pure coincidence their versions are in sync now.

Reply:
Got it!